test/functional/collection_tests.js

```diff
@@ -50,10 +59,7 @@ describe('Collection', function() {
               // Verify that all the result are correct coming back (should contain the value ok)
               expect(collection.collectionName).to.equal('test_collection_methods4');
               // Let's check that the collection was created correctly
-              db.listCollections().toArray(function(err, documents) {
-                let found = false;
-                if (documents.name === 'integration_tests_.test_collection_methods') found = true;
-                expect(found).to.be.true;
+              db.listCollections().toArray((err, documents) => {
```

Comment:
Do we still need this?

Reply:
It's testing the db.dropCollection function on line 71

Reply:
I think we need this test, but I think this `listCollections` is left over from a previous edit. If you look at the diff, it seems like the test does `db.createCollection('test_collection_methods4'` then `db.renameCollection(`, but the current version has this `listCollections` in between those steps..

Reply:
removed listCollections line.